### PR TITLE
Implement 'math ceiling' command

### PR DESCRIPTION
**Pull request by rwaskiewicz**
Implementation of `math ceiling` command for #115

### CODE REVIEW

File: src/Commands/MathCeilingCommand.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { CommandNames } from "./CommandNames";
+import { CommandMetadata } from "./Metadata/CommandMetadata";
+import { SingleParameter } from "./Metadata/Parameters/SingleParameter";
+import { NativeCallCommand } from "./NativeCallCommand";
+
+/**
+ * Rounds a number up to the nearest integer
+ */
+export class MathCeilingCommand extends NativeCallCommand {
+  /**
```
Comment (JoshuaKGoldberg) [inline src/Commands/MathCeilingCommand.ts]: Stylistic nitpick: these should be indented with 4 spaces, not 2. I'll file a separate issue to look into why lint rules aren't catching that...

File: src/Commands/MathCeilingCommand.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { Import } from "../Languages/Imports/Import";
+import { NativeCallProperties } from "../Languages/Properties/NativeCallProperties";
+import { CommandNames } from "./CommandNames";
+import { CommandMetadata } from "./Metadata/CommandMetadata";
+import { SingleParameter } from "./Metadata/Parameters/SingleParameter";
+import { NativeCallCommand } from "./NativeCallCommand";
+
+/**
+ * Rounds a number up to the nearest integer
```
Comment (JoshuaKGoldberg) [inline src/Commands/MathCeilingCommand.ts]: Stylistic nitpick: please end this comment with a period?

File: src/Commands/BannedKeywordsBag.ts [hunk 3]
```diff
@@ -37,6 +37,7 @@ export class BannedKeywordsBag {
             byte: true,
             case: true,
             catch: true,
+            ceiling: true,
```
Comment (JoshuaKGoldberg) [inline src/Commands/BannedKeywordsBag.ts]: Why is this a banned keyword? Is there a language that has `ceiling` as a keyword? If not, we can leave it out of here.

File: src/Commands/MathCeilingCommand.ts (hunk 2 above)
Comment (rwaskiewicz) [reply src/Commands/MathCeilingCommand.ts]: Np - added in 4730206d82c3ecfb9b5605e5af2b6fcbe5aca8eb

File: src/Commands/MathCeilingCommand.ts (hunk 1 above)
Comment (rwaskiewicz) [reply src/Commands/MathCeilingCommand.ts]: Updated in 4730206d82c3ecfb9b5605e5af2b6fcbe5aca8eb

File: src/Commands/BannedKeywordsBag.ts (hunk 3 above)
Comment (rwaskiewicz) [reply src/Commands/BannedKeywordsBag.ts]: Ah! Sorry about that, that was leftover from when I was still tinkering around with the project.  Removed in 4730206d82c3ecfb9b5605e5af2b6fcbe5aca8eb.  Thanks for the feedback!

**Comment by JoshuaKGoldberg** [comment]
LGTM, thanks!

**Pull request merged**